packages/accordions/src/styled/accordion/StyledPanel.ts

```diff
@@ -17,38 +22,56 @@ interface IStyledPanel {
 
 const COMPONENT_ID = 'accordions.panel';
 
-const paddingStyles = (props: IStyledPanel & ThemeProps<DefaultTheme>) => {
-  const { base } = props.theme.space;
+const colorStyles = (props: IStyledPanel & ThemeProps<DefaultTheme>) => {
+  const { theme, isBare } = props;
+
+  return css`
+    border-bottom-color: ${isBare ? 'transparent' : getColor('neutralHue', 300, theme)};
+  `;
+};
+
+const sizeStyles = (props: IStyledPanel & ThemeProps<DefaultTheme>) => {
+  const { theme, isCompact, isExpanded } = props;
+  const { base } = theme.space;
   let paddingTop = base * 2;
   let paddingHorizontal = base * 5;
   let paddingBottom = base * 8;
 
-  if (props.isCompact) {
+  if (isCompact) {
     paddingTop = base * 2;
     paddingHorizontal = base * 3;
     paddingBottom = base * 4;
   }
 
-  if (props.isExpanded === false) {
+  if (isExpanded === false) {
     paddingTop = 0;
     paddingBottom = 0;
   }
 
   return css`
-    transition: ${props.isAnimated && 'padding 0.25s ease-in-out'};
+    grid-template-rows: ${isExpanded ? 1 : 0}fr;
+    border-bottom: ${theme.borders.sm};
     padding: ${paddingTop}px ${paddingHorizontal}px ${paddingBottom}px;
+    line-height: ${getLineHeight(base * 5, theme.fontSizes.md)};
+    font-size: ${theme.fontSizes.md};
   `;
 };
 
+/**
+ * 1. reverse order of `sizeStyles` and `colorStyles` to give `border-bottom-color` styling precedence
+ */
```

Comment:
With the previous recommendation, I think we can remove this comment – especially since the updated order now honors the documented [rules](https://github.com/zendeskgarden/react-components/blob/main/docs/api.md#rules).

Reply:
Good call, that was a mixup on my end.